chat/public/css/chat.css

```diff
@@ -30,6 +30,45 @@ body.view-in-course .course-wrapper.chromeless {
     height: 100vh;
 }
 
+.chat-block .subject {
+  width: 100%;
+  z-index: 999;
```

Comment:
Hmm, yeah, your're right.

I just added this to be sure it would be on top, but didn't check if it was actually needed.

I'll remove it from the .css